test/aws.storage.test.js

```diff
@@ -21,145 +21,156 @@ redisStub.createClient = function() {
     hmset: hmset,
     expire: expire,
     del: del
-  }
-}
+  };
+};
 
-let fsStub = {};
+const fsStub = {};
 fsStub.statSync = sinon.stub();
 fsStub.createReadStream = sinon.stub();
 fsStub.createWriteStream = sinon.stub();
 fsStub.unlinkSync = sinon.stub();
 
-let logStub = {};
+const logStub = {};
 logStub.info = sinon.stub();
 logStub.error = sinon.stub();
 
-let s3Stub = {};
+const s3Stub = {};
 s3Stub.headObject = sinon.stub();
 s3Stub.getObject = sinon.stub();
 s3Stub.upload = sinon.stub();
 s3Stub.deleteObject = sinon.stub();
 
-let awsStub = {
-    S3: function() {
-        return s3Stub;
-    }
-}
+const awsStub = {
+  S3: function() {
+    return s3Stub;
+  }
+};
 
 const storage = proxyquire('../server/storage', {
-  'redis': redisStub,
-  'fs': fsStub,
-  './log.js': function() { return logStub },
+  redis: redisStub,
+  fs: fsStub,
+  './log.js': function() {
+    return logStub;
+  },
   'aws-sdk': awsStub
 });
 
 describe('Testing Length using aws', function() {
   it('Filesize returns properly if id exists', function() {
-    s3Stub.headObject.callsArgWith(1, null, {ContentLength: 1});
-    return storage.length('123')
-                  .then(reply => assert.equal(reply, 1))
-                  .catch(err => assert.fail())
-  })
+    s3Stub.headObject.callsArgWith(1, null, { ContentLength: 1 });
+    return storage
+      .length('123')
+      .then(reply => assert.equal(reply, 1))
+      .catch(err => assert.fail());
+  });
 
   it('Filesize fails if the id does not exist', function() {
     s3Stub.headObject.callsArgWith(1, new Error(), null);
-    return storage.length('123')
-                  .then(reply => assert.fail())
-                  .catch(err => assert(1))
-  })
+    return storage
+      .length('123')
+      .then(_reply => assert.fail())
+      .catch(err => assert(1));
+  });
 });
 
 describe('Testing Get using aws', function() {
-
   it('Should not error out when the file exists', function() {
-    let spy = sinon.spy();
+    const spy = sinon.spy();
     s3Stub.getObject.returns({
       createReadStream: spy
     });
 
     storage.get('123');
     assert(spy.calledOnce);
-  })
+  });
 
   it('Should error when the file does not exist', function() {
-    let err = function() { throw new Error(); }
-    let spy = sinon.spy(err);
+    const err = function() {
+      throw new Error();
+    };
+    const spy = sinon.spy(err);
     s3Stub.getObject.returns({
       createReadStream: spy
     });
 
     assert.equal(storage.get('123'), null);
     assert(spy.threw());
-  })
+  });
 });
 
 describe('Testing Set using aws', function() {
   beforeEach(function() {
     expire.reset();
-  })
+  });
 
   after(function() {
     crypto.randomBytes.restore();
-  })
+  });
 
   it('Should pass when the file is successfully uploaded and no bitly key', function() {
     conf.bitly_key = null;
-    const buf =  new Buffer(10);
+    const buf = Buffer.alloc(10);
```

Comment:
Note that `new Buffer(<size>)` seems to be deprecated: https://nodejs.org/api/buffer.html#buffer_new_buffer_size